src/types.ts

```diff
@@ -539,7 +539,8 @@ export function detectMapType(
 
 export function requestType(ctx: Context, methodDesc: MethodDescriptorProto): Code {
   let typeName = messageToTypeName(ctx, methodDesc.inputType);
-  if (methodDesc.clientStreaming) {
+  const { options } = ctx;
+  if (options.nestJs && methodDesc.clientStreaming) {
```

Comment:
I think this fixes the compile error, but in a pretty indirect way, right? Like the `*.proto` file is saying "the client can submit a stream of messages" and this is basically saying "we won't do that"...which, compiles, but does it actually work?

Like looking at the grpc-web examples, if a method is `stream Cred` we can't just write a `Cred` and call it good, we need to tell the server "we _could_ have sent more `Cred`s but we're not", i.e. by calling `finishSend`:

https://github.com/improbable-eng/grpc-web/blob/master/client/grpc-web/src/client.ts#L304

So this is less of "just fix the compile error" and more that I think we need to actually support client-to-server / request streaming. Until then, ts-proto just won't work with `*.proto` files that use `stream Cred`.

Maybe we could like skip putting those methods in the output until it's supported, so that if you have a `*.proto` file that has those, but you don't need to call them, you'd at least get output that would let you use the other methods.

Reply:
Yes, you are right, this does just fix the compile error. I don't have rpc services with a backend setup right now test; though; from what [improbable says](https://github.com/improbable-eng/grpc-web#client-side-streaming) , it's not supported. 

Omitting entirely the `stream Cred` methods sounds like a good plan; stay tuned